### PR TITLE
Add issue templates

### PR DESCRIPTION
**Pull request by timmydoza**
This PR adds issue templates to the repo. This PR was created with Github's template creator (which must be why it isn't using the PR template).

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: ''
+assignees: timmydoza
```
Comment (aaalaniz) [inline .github/ISSUE_TEMPLATE/bug_report.md]: remove assignees

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
+---
+name: Feature request
+about: Suggest an idea for this project
+title: ''
+labels: ''
+assignees: timmydoza
```
Comment (aaalaniz) [inline .github/ISSUE_TEMPLATE/feature_request.md]: remove assignees

Comment (aaalaniz) [reply .github/ISSUE_TEMPLATE/feature_request.md]: add label "feature request"

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 1 above)
Comment (aaalaniz) [reply .github/ISSUE_TEMPLATE/bug_report.md]: add label "bug"

File: .github/ISSUE_TEMPLATE/feature_request.md (hunk 2 above)
Comment (timmydoza) [reply .github/ISSUE_TEMPLATE/feature_request.md]: Added label 'feature' to correspond with the label:
image

**Pull request merged**